cmd/build.go

```diff
@@ -225,7 +225,7 @@ func getLabels(config *RootCommandConfig) (map[string]string, error) {
 		return labels, projectConfigErr
 	}
 
-	configLabels, err := getConfigLabels(*projectConfig, ".appsody-config.yaml")
+	configLabels, err := getConfigLabels(*projectConfig, ".appsody-config.yaml", config)
```

Comment:
By only using `log *LoggingConfig` in the functions below, this becomes `config.LoggingConfig`